### ISSUE TITLE
resolveRouteComponents is not defined

### ISSUE THREAD

**Issue by Vout** [nuxt/nuxt.js#7587]
### Version

[v2.13.0](https://github.com/nuxt.js/releases/tag/v2.13.0)

### Reproduction link

[https://github.com/Vout/purvis-frontend](https://github.com/Vout/purvis-frontend)

### Steps to reproduce

Upgraded project from 2.12.2 to 2.13

### What is expected ?

Normal navigation between routes without the error message flashing by.

### What is actually happening?

When navigating between routes the following message flashes by,

```
resolveRouteComponents is not defined
An error occurred while rendering the page. Check developer tools console for details.
```

The console however stays empty same goes for the commandline, no errors at all.

Running in dev or production mode doesn't matter, it always seems to be there.

It is mentioned in this github issue but i don't think it's related to the nuxt/content part, https://github.com/nuxt/content/issues/123

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c10831">#c10831</a>)</em></sub></div>

**Comment by Vout** [comment]
Setting up a new project, copy all the files to the new project and updating the nuxt.config.js seems have fixed the issue however it shouldn't be the "way to go" to fix this error.

**Comment by pi0** [comment]
Hi! This should be fixed with [v2.13.1](https://github.com/nuxt/nuxt.js/releases/tag/v2.13.1).
